learn/contributing/contributing_to_docs.md

```diff
@@ -0,0 +1,91 @@
+# Documentation
+
+This documentation website is hosted in a [public GitHub repository](https://github.com/meilisearch/documentation). It is built with [VuePress](https://vuepress.github.io), written in [MarkDown](https://www.markdownguide.org/getting-started/), and deployed on [Netlify](https://www.netlify.com).
+
+## Our Documentation Philosophy
+
+Our documentation aims to be:
+
+- **Efficient**. We don't want to waste anyone's time.
+- **Simple**. Reading the documentation shouldn't require native English or an advanced degree.
+- **Thorough**. Behavior that isn't documented or is documented incorrectly, we consider broken.
+- **Open source**. This is a resource by MeiliSearch users, for MeiliSearch users. [We welcome outside contributions](#how-to-contribute).
+
+## How to Contribute?
+
+### Issues
+
+On the MeiliSearch documentation team, [GitHub Issues](https://github.com/meilisearch/documentation/issues/new/choose) are the primary tool we use to track tasks. They're like elements in our to-do list. Here are some good uses for issues:
+
+- A detailed bug report
+- A request for a new technical feature (e.g. dark mode, versioning, etc.)
+- A request for new content (e.g. a tutorial, guide, or explanation)
+- A helpful suggestion for the future of the documentation
+
+We love issues, because they help us do our jobs better. Nine times out of ten, the most useful contribution isn't a flashy [pull request](#pull-requests) that needs to be read and reviewed—it's a simple GitHub issue that points out a problem and proposes a solution.
```

Comment:
I would like to keep some aspect of this message in there, because encouraging issues over PRs will make my job easier. Maybe I can rephrase it so I'm not dissing PRs?

```suggestion
We love issues, because they help us do our jobs better. Nine times out of ten, the most useful contribution is a simple GitHub issue that points out a problem and proposes a solution.
```